be/user.go

```diff
@@ -45,3 +53,37 @@ func UpdateUser(c *gin.Context) {
 	}
 	c.Status(http.StatusOK) // 200
 }
+
+func UpdatePrivacyAndTOC(c *gin.Context) {
```

Comment:
I thought we agreed to have one file per endpoint, no?

Reply:
I moved it into its own file